packages/core/crates/core-protocol/src/msg/processor.rs

```diff
@@ -120,9 +122,260 @@ where
     }
 }
 
-pub enum PacketType {
-    Final(Packet, Option<Acknowledgement>),
-    Forward(Packet, Option<Acknowledgement>, PeerId, PeerId),
+#[async_trait::async_trait(? Send)]
```

Comment:
Do we still need it to be non-send? If the WASM barrier has moved further, it might be made `Send` already

Reply:
The DB itself is `? Send`, once that is `Send`, all traits can be updated.